src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -110,12 +110,19 @@ namespace winrt::TerminalApp::implementation
     {
         const auto selected = _filteredActionsView().SelectedIndex();
         const int numItems = ::base::saturated_cast<int>(_filteredActionsView().Items().Size());
-        // Wraparound math. By adding numItems and then calculating modulo numItems,
-        // we clamp the values to the range [0, numItems) while still supporting moving
-        // upward from 0 to numItems - 1.
-        const auto newIndex = ((numItems + selected + (moveDown ? 1 : -1)) % numItems);
-        _filteredActionsView().SelectedIndex(newIndex);
-        _filteredActionsView().ScrollIntoView(_filteredActionsView().SelectedItem());
+
+        // Do not try to select an item if
+        // - the list is empty
+        // - if no item is selected and "up" is pressed
```

Comment:
Question: the comment refers to 'up', but the if statement moveDown.  What am I missing?

Reply:
Upon down we go to 0, which is a desired behavior in CommandLine mode. I agree that it is not very explicit and probably worth commenting

Reply:
Sorry - for confusing comment
* In the comment: don't try to select if up is pressed
* In the code: try to select if down is pressed

!moveDown = up